s3tests_boto3/functional/test_s3.py

```diff
@@ -376,6 +545,57 @@ def test_bucket_list_delimiter_none():
     eq(keys, key_names)
     eq(prefixes, [])
 
+@attr(resource='bucket')
+@attr(method='get')
+@attr(assertion='unspecified delimiter defaults to none')
+@attr('list-objects-v2')
+def test_bucket_listv2_delimiter_none():
+    key_names = ['bar', 'baz', 'cab', 'foo']
+    bucket_name = _create_objects(keys=key_names)
+    client = get_client()
+
+    response = client.list_objects_v2(Bucket=bucket_name)
+    # putting an empty value into Delimiter will not return a value in the response
+    eq('Delimiter' in response, False)
+
+    keys = _get_keys(response)
+    prefixes = _get_prefixes(response)
+    eq(keys, key_names)
+    eq(prefixes, [])
+
+@attr('list-objects-v2')
+def test_bucket_listv2_owner_notempty():
```

Comment:
@albIN7 could you change `owner` in these test names to `fetchowner`.

Reply:
Done